src/EditorFeatures/Core.Wpf/Classification/ClassificationTypeFormatDefinitions.cs

```diff
@@ -731,6 +731,24 @@ public XmlDocCommentTextFormatDefinition()
         }
         #endregion
 
+        [Export(typeof(EditorFormatDefinition))]
+        [ClassificationType(ClassificationTypeNames = ClassificationTypeNames.TestCode)]
+        [Name(ClassificationTypeNames.TestCode)]
+        [Order(After = ClassificationTypeNames.StringLiteral)]
+        [Order(After = ClassificationTypeNames.VerbatimStringLiteral)]
+        [UserVisible(true)]
+        [ExcludeFromCodeCoverage]
```

Comment:
Do we even have code coverage configured?

Reply:
no.  but this just follows the pattern of everything else :)